source/specs.txt

```diff
@@ -5,179 +5,180 @@ MongoDB Drivers Specs
 .. default-domain:: mongodb
 
 MongoDB produces specifications for features of MongoDB and for the
-MongoDB drivers. The following table lists all public specs and
-what they apply to.
+MongoDB drivers, located in our
+`Github repository <https://github.com/mongodb/specifications>`__
```

Comment:
I would make the text of this link more specific since I am not clear whether this opens an informational page on "Github repository" or the actual repository.
"MongoDB publishes official specifications for features of MongoDB and the MongoDB drivers. For a complete list of specifications, see the `mongodb/specifications Github repository <...>`.